flight-server/src/main/java/org/m4d/adp/flight_server/ExampleFlightServer.java

```diff
@@ -37,10 +37,18 @@ public void awaitTermination() throws InterruptedException {
      *  Main method starts the server listening to localhost:12233.
      */
     public static void main(String[] args) throws Exception {
-        if ((args.length < 1) || (args.length > 2)) {
-            System.out.println("Need single argument: either 'example' or 'relay'");
+        if ((args.length != 3) && (args.length != 6)) {
+            System.out.println("Arguments are either:");
+            System.out.println("\texample host port");
+            System.out.println("\trelay transformation(true/false) host port remote_host remote_port");
```

Comment:
There must be a better way to do that (e.g., subcommands)

Reply:
There is, and @Mohammad-nassar10 is already working on it. Stay tuned...